card/lib/card/auth.rb

```diff
@@ -33,13 +46,12 @@ def encrypt password, salt
 
       # find accounted by email
```

Comment:
...which implies you already figured out my last comment :)  thanks for spotting that.